README.md

```diff
@@ -1,6 +1,17 @@
 # go-square
 
-`go-square` is a Go library designed to provide utility functions for data square operations.
+[![Go Reference](https://pkg.go.dev/badge/github.com/celestiaorg/go-square.svg)](https://pkg.go.dev/github.com/celestiaorg/go-square)
+
+`go-square` is a Go module that provides types and utilities for interacting with the original data square in a Celestia block.
```

Comment:
```suggestion
`go-square` is a Go module that provides data structures and utilities for manipulation and interactions with data squares in Celestia's network. Data square is a special form of block serialization in Celestia blockchain.
```

The way I would phrase it. After, we could also link to specs.

Reply:
Note if we remove "original" from this line then we should add a separate sentence to explain the difference between original and extended because the issue this PR is trying to close was particularly confused about original vs extended.

Reply:
You suggestion SGTM :+1:

Reply:
https://github.com/celestiaorg/go-square/pull/34/commits/bb14a7ff02a267da1143fde542c9dc96719a9386